provider/oci/instance.go

```diff
@@ -44,6 +44,8 @@ type vnicWithIndex struct {
 }
 
 var _ instance.Instance = (*ociInstance)(nil)
+var maxPollIterations = 30
+var pollTime time.Duration = 10
```

Comment:
Did we want the poll time to be 10 nanoseconds?
I think you need some sort of 10*time.Second, 10*time.Millisecond?

I see that later on you are multiplying it time time.Second, but since it is a time.Duration, it would be clearer to do the multiplication here, and just treat it as a duration later.

Reply:
changed